src/components/30-organisms/modal/index.scss

```diff
@@ -0,0 +1,79 @@
+.o-modal {
+  display: none;
+  width: 100%;
+  height: 100%;
+
+  position: fixed;
+  // Highest z-index on the axa website
+  z-index: 3000;
+  left: 0;
+  top: 0;
+
+  background-color: rgba(0, 0, 0, 0.5);
+
+  @include breakpoint($mediaquery-xs-up) {
+    align-items: center;
+  }
+
+  &--open {
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+  }
+
+  &__upper-close-container {
+    display: flex;
+    z-index: 1;
+    margin: 10px 10px 0px;
+    box-shadow: 0px 1px 10px 0px rgba(0, 0, 0, 0.15);
+    justify-content: flex-end;
+    border-radius: 8px 8px 0px 0px;
+    background-color: white;
+
+    @include breakpoint($mediaquery-xs-up) {
+      width: 60%;
+      padding: 0px;
+      margin: 0px;
+      background-color: white;
+    }
+
+    &-button {
+      cursor: pointer;
+      border: none;
+      background: transparent;
+      outline: none;
+      color: inherit;
+      font: inherit;
+      line-height: normal;
+      margin: 8px;
+      height: 40px;
+      padding: 8px;
+      
+      svg {
+        height: 24px;
+        width: 24px;
+      }
+    }
+  }
+
+  &__content {
+    position: relative;
+    overflow: auto;
+    padding: 0px 28px 28px;
+    margin: 0px 10px 10px;
+    height: 100%;
+    background-color: white;
+    border-radius: 0px 0px 8px 8px;
+
+    @include breakpoint($mediaquery-xs-up) {
+      overflow: auto;
+      box-sizing: border-box;
+      padding: 0px 40px 40px;
+      width: 60%;
+      margin: 0px;
+      max-height: 50%;
+      background-color: white;
```

Comment:
please use our variables everywhere in your scss files. You can find them here `src/components/00-materials/styles/00-colors.scss`

For example white is:
$color-prim-white